backend/src/apiserver/resource/resource_manager.go

```diff
@@ -252,6 +252,8 @@ func (r *ResourceManager) CreatePipeline(name string, description string, pipeli
 		return nil, util.Wrap(err, "Create pipeline failed")
 	}
 
+	newPipeline.DefaultVersion.
```

Comment:
seems this PR is hold. will you continue?

Reply:
I will pick this up and finalize it, actually forgot about it @rmgogogo